### DISCUSSION TITLE
Curso FullStack .NET - CORS

### DISCUSSION THREAD

**Discussion by danilobicas-unoeste** [balta-io/forum#627] [Tira Dúvidas] [answered]
Não consigo acessar os endpoints... Sempre recebo a mensagem de erro (no console das ferramentas de desenvolvedor do edge): " _http://localhost:5274/v1/identity/manage/info' from origin 'http://localhost:5064' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled._" Está configurado o backend com a url correta e o frontend também..... Agradeço a quem puder ajudar.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @danilobicas-unoeste, como vai?

É o problema de CORS que menciono no curso, você deve ter perdido alguma coisa durante as aulas.
Primeiro passo é revisar as aulas e comparar seu código com o código apresentado no curso, principalmente as aulas de CORS!

Caso não encontre o erro, posta aqui a parte da sua API, as extensions que construiu e o program.cs que damos uma olhada.
